react-events-and-state/src/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isClicked: true });
+  }
+
+  render() {
+    const clicked = this.state.isClicked;
+    // eslint-disable-next-line react/prop-types
```

Comment:
If you name your file with a `.jsx` extension instead of `.js` you won't need this line